pkgs/ffigen/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 12.1.0-wip
+
+- Code-gen the ObjC `id` type to `ObjCObjectBase` rather than `NSObject`, since
```

Comment:
Isn't this technically a breaking change?

Reply:
I think it's unlikely anyone will hit this, but yeah, it's technically a breaking change. I'll make it a major version bump anyway just to be safe.